homeassistant/components/lightwave/climate.py

```diff
@@ -0,0 +1,167 @@
+"""Support for LightwaveRF TRVs."""
+import logging
+
+from homeassistant.components.climate import (
+    DEFAULT_MAX_TEMP,
+    DEFAULT_MIN_TEMP,
+    HVAC_MODE_HEAT,
+    HVAC_MODE_OFF,
+    SUPPORT_TARGET_TEMPERATURE,
+    ClimateDevice,
+)
+from homeassistant.components.climate.const import CURRENT_HVAC_HEAT, CURRENT_HVAC_OFF
+from homeassistant.const import (
+    ATTR_BATTERY_LEVEL,
+    ATTR_TEMPERATURE,
+    CONF_NAME,
+    TEMP_CELSIUS,
+)
+
+from . import CONF_SERIAL, LIGHTWAVE_LINK
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Find and return LightWave lights."""
+    if not discovery_info:
+        return
+
+    trv = []
+    lwlink = hass.data[LIGHTWAVE_LINK]
+
+    for device_id, device_config in discovery_info.items():
+        name = device_config[CONF_NAME]
+        serial = device_config[CONF_SERIAL]
+        trv.append(LightwaveTrv(name, device_id, lwlink, serial))
+
+    async_add_entities(trv)
+
+
+class LightwaveTrv(ClimateDevice):
+    """Representation of a LightWaveRF TRV."""
+
+    def __init__(self, name, device_id, lwlink, serial):
+        """Initialize LightwaveTrv entity."""
+        self._name = name
+        self._device_id = device_id
+        self._state = None
+        self._max_temp = DEFAULT_MAX_TEMP
+        self._min_temp = DEFAULT_MIN_TEMP
```

Comment:
You do not need attributes for these values since they never change, it would make it easier to read if you just return the constant in the respective property.

Reply:
Fixed